bundles/org.eclipse.packagedrone.utils.deb.tests/src/org/eclipse/packagedrone/utils/deb/tests/BinaryPackageTest.java

```diff
@@ -10,36 +10,74 @@
  *******************************************************************************/
 package org.eclipse.packagedrone.utils.deb.tests;
 
+import static java.time.temporal.TemporalAdjusters.firstDayOfYear;
+
 import java.io.File;
+import java.io.FileNotFoundException;
 import java.io.FileOutputStream;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.function.Supplier;
 
 import org.eclipse.packagedrone.utils.deb.build.DebianPackageWriter;
 import org.eclipse.packagedrone.utils.deb.build.EntryInformation;
 import org.eclipse.packagedrone.utils.deb.control.BinaryPackageControlFile;
 import org.junit.Assert;
 import org.junit.Test;
 
+import com.google.common.hash.Hashing;
+
 public class BinaryPackageTest
 {
     @Test
-    public void test1 () throws IOException
+    public void test1 () throws IOException, InterruptedException
     {
-        final File file = File.createTempFile ( "test-", ".deb" );
+        final File file1 = File.createTempFile ( "test-1-", ".deb" );
+        final File file2 = File.createTempFile ( "test-2-", ".deb" );
+
+        final long startOfYear = LocalDate.now ().with ( firstDayOfYear () ).toEpochDay () * 24 * 60 * 60 * 1000; // 2015-11-23
+        final Supplier<Instant> timestampProvider = new Supplier<Instant> () {
```

Comment:
Please use a lambda expression for that `timestampProvider = () -> testTimestamp;`

Also I think that there is a more readable way to generate a timestamp in Java 8.

Like:

```
Instant testTimestamp = ZonedDateTime().of ( 2015, Month.November, 23, 0, 0, 0, ZoneOffset.UTC).toInstant();`
```